src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.cs

```diff
@@ -119,6 +119,221 @@ public new abstract class Arm64 : AdvSimd.Arm64
         public static unsafe Vector<ulong> CreateTrueMaskUInt64([ConstantExpected] SveMaskPattern pattern = SveMaskPattern.All) => CreateTrueMaskUInt64(pattern);
 
 
+        ///  CreateWhileLessThanMask16Bit : While incrementing scalar is less than
+
+        /// <summary>
+        /// svbool_t svwhilelt_b16[_s32](int32_t op1, int32_t op2)
+        ///   WHILELT Presult.H, Wop1, Wop2
+        /// </summary>
+        public static unsafe Vector<ushort> CreateWhileLessThanMask16Bit(int left, int right) => CreateWhileLessThanMask16Bit(left, right);
```

Comment:
nit: I would have the following order:
-  CreateWhileLessThanMask8Bit
- CreateWhileLessThanMask16Bit
- CreateWhileLessThanMask32Bit
- CreateWhileLessThanMask64Bit

Reply:
If we're keeping to strict machine generate alphabetical, then 8bit would go at the end. Happy to move it if not.

Reply:
The existing tooling for auto generating ref assemblies expects it in alphabetical order

so `8` comes after `64`